README.md

```diff
@@ -83,6 +83,10 @@ Runs linting against documentation using [silverstripe/documentation-lint](https
 Runs PHPunit if the `phpunit.xml` or `phpunit.xml.dist` config file is available. Default is true, disable PHPunit tests with:
 `phpunit: false`
 
+##### Skip PHPUnit suites
+Skips specific PHPUnit suites in the dynamically generated matrix. Default is to run all suites. Configure with:
+`phpunit_skip_suites: suite-to-skip, another-to-skip`
```

Comment:
```suggestion
`phpunit_skip_suites: suite-to-skip,another-to-skip`
```
Should not include a space after the comma when comma delimiting, we're not making sentences :-)

Reply:
With a space is more readable, but it doesn't really matter so I'll make the change to avoid ping pong

Reply:
Done